Elias/exercise-4.js

```diff
@@ -0,0 +1,42 @@
+//Variables
+let apples = 20;
+let oranges = 30;
+let mangos = 5;
+
+// Are apples and oranges equal?
+console.log(apples === oranges); //==> false
+
+//Are apples and oranges unequal?
+console.log(apples !== oranges); //==> true
+
+// are apples greater than oragnes?
+console.log(apples > oranges); //==> false
+
+// are apples less than oranges?
```

Comment:
Here also try to check if they are EQUAL(less or equal) 
means <= instead of <

Reply:
The third question only states if it is greater than, so I fulfilled the task. Right after that however, I made a mistake and there I have to add <=. Thanks for checking out my code.